src/App.tsx

```diff
@@ -1,16 +1,24 @@
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+import { Movie } from './types/Movie';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => {
+export const App: React.FC = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const onAdd = (movieToAdd: Movie): void => {
+    setMovies((prevMovies) => [...prevMovies, movieToAdd]);
+  };
+
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={movies} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie onAdd={onAdd} />
```

Comment:
So here you just have to change your function name and leave prop name
```suggestion
        <NewMovie onAdd={addMovie} />
```

Reply:
Done